src/section-notation-for-intervals.ptx

```diff
@@ -166,11 +162,118 @@
                     </sidebyside>
                 </sbsgroup>
 
-                <p>Also we allow <m>a</m> or <m>b</m> to be the symbols <m>\infty</m> or <m>-\infty</m>. If these symbols are used then the interval extends forever in one direction. Wherever these symbols are used, there has to be a round parenthesis, not a square bracket, since the interval won't actually include these (non-) numbers.</p>
-            </statement>
+<!--                 <p>Also we allow <m>a</m> or <m>b</m> to be the symbols <m>\infty</m> or <m>-\infty</m>. If these symbols are used then the interval extends forever in one direction. Wherever these symbols are used, there has to be a round parenthesis, not a square bracket, since the interval won't actually include these (non-) numbers.</p>
+ -->            </statement>
         </definition>
 
-        <example>
+<!--TODO: change this so it is not identical to exercise below-->
```

Comment:
Can you say more what you mean here? Like what is identical to what, maybe using line numbers to say. Also, is something about the source code identical, or the end-output the reader sees?